mime-parse/src/lib.rs

```diff
@@ -132,6 +132,26 @@ impl Mime {
         self.semicolon().is_some()
     }
 
+    #[inline]
+    pub fn without_params(self) -> Self {
+        let semicolon = match self.semicolon() {
+            None => return self,
+            Some(i) => i,
+        };
+
+        let mut mtype = self;
+        mtype.params = ParamSource::None;
+        mtype.source = match mtype.source {
+            Source::Atom(_, s) => {
+                Atoms::intern(&s[..semicolon], mtype.slash, InternParams::None)
+            },
+            Source::Dynamic(s) => {
+                Source::Dynamic(s[..semicolon].to_string())
```

Comment:
That which made it dynamic may have been the parameters, so could probably just try to intern either way, no?

Reply:
Ah, silly me, that's indeed likely in most cases :blush:.